tests/test_aws.py

```diff
@@ -370,6 +370,26 @@ def test_aws_get_running_apps(monkeypatch):
     boto.assert_called_with('ec2', region_name=REGION)
 
 
+def test_aws_get_running_apps_existing(monkeypatch):
```

Comment:
You are not making the actual call here to ``aws.get_running_apps`` and asserting the result 😉